runtime/common/src/lib.rs

```diff
@@ -77,7 +77,7 @@ parameter_types! {
 	/// Minimum amount of the multiplier. This value cannot be too low. A test case should ensure
 	/// that combined with `AdjustmentVariable`, we can recover from the minimum.
 	/// See `multiplier_can_grow_from_zero`.
-	pub MinimumMultiplier: Multiplier = Multiplier::saturating_from_rational(1, 1_000_000_000u128);
+	pub MinimumMultiplier: Multiplier = Multiplier::saturating_from_rational(1, 1_000u128);
```

Comment:
Even if this is the correct value, I might prefer we reduce it by 1,000 over 2 upgrades than by 1,000,000 at once :)

Reply:
cowboy prevention system activated 🤠 

fair enough.